DO NOT MERGE YET - including ZFA in go-lego

cmungall opened the pull request.

cmungall committed:
DO NOT MERGE YET - including ZFA in go-lego

cmungall commented:
Required for https://github.com/geneontology/noctua/issues/541

balhoff commented on src/ontology/extensions/go-lego.owl:
```diff
@@ -17,6 +17,8 @@ Import(<http://purl.obolibrary.org/obo/eco.owl>)
 Import(<http://purl.obolibrary.org/obo/wbbt.owl>)
 Import(<http://purl.obolibrary.org/obo/emapa.owl>)
 Import(<http://purl.obolibrary.org/obo/uberon/bridge/uberon-bridge-to-emapa.owl>)
+Import(<http://purl.obolibrary.org/obo/zfa.owl>)
```
Possibly instead: `<http://purl.obolibrary.org/obo/upheno/imports/zfa_import.owl>`

Along with the Uberon bridge.

balhoff committed:
Import only ZFA, not Uberon-bridge-to-ZFA

ZFA has been changed to be consistent with material/immaterial entity distinction; further alignment with Uberon will require more time. See https://github.com/geneontology/noctua/issues/541

balhoff commented:
I am going to go ahead and merge this without the Uberon bridge. This way ZFA can be used in models. We should consider making a small bridge from ZFA to CARO to make use of some disjointness without bridging to Uberon.
